src/gui/mzroll/tabledockwidget.cpp

```diff
@@ -1765,6 +1765,59 @@ void TableDockWidget::showIntegrationSettings()
   log.exec();
 }
 
+void TableDockWidget::moveSelectedRows(QString destinationTableName) {
+
+  QString sourceTableName = this->titlePeakTable->text();
+  QList<QPointer<TableDockWidget>> peaksTableList = _mainwindow->getPeakTableList();
+  QList<shared_ptr<PeakGroup>> selected = getSelectedGroups();
+  QString bookmarkTableName = _mainwindow->bookmarkedPeaks->titlePeakTable->text();
+
+  if(bookmarkTableName == destinationTableName) {
+    BookmarkTableDockWidget* destinationPeakTable = _mainwindow->bookmarkedPeaks;
+    for(auto group : selected) {
+      destinationPeakTable->addPeakGroup(group.get());
+    }
+    destinationPeakTable->showAllGroups();
+    destinationPeakTable->setVisible(true);
+  } else {
+    TableDockWidget* destinationPeakTable = nullptr;
+    for (auto table : peaksTableList) {
+        if (table->titlePeakTable->text() == destinationTableName) {
+            destinationPeakTable = table;
+        }
+        if(destinationPeakTable != nullptr) {
+          break;
+        }
+    }
+
+    for(auto group : selected) {
+      destinationPeakTable->addPeakGroup(group.get());
```

Comment:
@MdAzmatAli, I think this might lead to a segment fault. In the case where the _destinationPeakTable_ is 'nullptr' even after for loop. 
So, for the safer side, we can add a check on line 1792 -
`if (!destinationPeakTable)
         return;
`

Reply:
I think it will not happen but I have added it for safer side @sakshikukreja14